addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -600,12 +600,15 @@ var ListRenderer = BasicRenderer.extend({
 
         if (record.res_id) {
             // TODO this should be moved to a handler
-            $button.on("click", function (e) {
-                e.stopPropagation();
+            const debouncedClick = _.debounce(() => {
                 self.trigger_up('button_clicked', {
                     attrs: node.attrs,
                     record: record,
                 });
+            }, 500);
```

Comment:
set the `immediate` argument to `true` (see http://underscorejs.org/#debounce), to execute the handler directly. That way, no need to do weird stuff in tests, as the callback is still executed in a synchronous way.

Reply:
@aab-odoo Thanks, I didn't check ```immediate``` argument, Thanks, I did changes accordingly.